pkg/api/validation/validation.go

```diff
@@ -114,7 +131,7 @@ func nameIsDNS952Label(name string, prefix bool) (bool, string) {
 	if util.IsDNS952Label(name) {
 		return true, ""
 	}
-	return false, "name must be lowercase letters, numbers, and dashes"
+	return false, name + " must be lowercase letters, numbers, and dashes."
```

Comment:
Just replace this whole string with DNS952LabelErrorMsg, which should have the same structure as all the rest